static/js/textMarkInfoDialog.js

```diff
@@ -140,7 +161,7 @@ textMarkInfoDialog.prototype._showTextMarkInfoDialog = function(
 
   this._keepCurrentRepSelection(function() {
     var dialog = self.infoDialog;
-    self._fillTextMarkContentOnInfoDialog();
+    self._fillTextMarkContentOnInfoDialog(self.addAdditionalElementsOnInfoDialog);
```

Comment:
como vc guarda o `addAdditionalElementsOnInfoDialog`, nem precisaria passá-lo como parâmetro, certo? Daria para pegá-lo direto on de vc for usar...